### PR TITLE
Site db client

### PR DESCRIPTION
**Pull request by jha2**
implemented fix for the test SiteDB_t.py

### CODE REVIEW

**Comment by jha2** [comment]
Please test it.

**Comment by jha2** [comment]
Please test it.

**Comment by ericvaandering** [comment]
This is not what we discussed. Make a commit with just the SiteDB emulator changes and the SiteDB emulator tests.

File: test/python/WMCore_t/Services_t/SiteDB_t/SiteDB_t.py [hunk 1]
```diff
@@ -117,7 +117,32 @@ def testPNNtoPSN(self):
     def testCMSNametoList(self):
         result = self.mySiteDB.cmsNametoList("T1_US*", "SE")
         self.assertItemsEqual(result, [u'cmssrm.fnal.gov', u'cmssrmdisk.fnal.gov'])
-        
+
+    def testCheckAndConvertSENameToPNN(self):
+        """
+        Test the conversion of SE name to PNN for single and multiple sites/PNNs using checkAndConvertSENameToPNN
+        """
```
Comment (ericvaandering) [inline test/python/WMCore_t/Services_t/SiteDB_t/SiteDB_t.py]: Get rid of this file

File: src/python/WMQuality/Emulators/SiteDBClient/SiteDB.py [hunk 2]
```diff
@@ -303,21 +312,18 @@ def PSNtoPNNMap(self, psn_pattern=''):
...
+        if isinstance(seNameOrPNN, basestring):
             seNameOrPNN = [seNameOrPNN]
-        
         newList = []
         for se in seNameOrPNN:
             if not pnn_regex.match(se):
                 newList.extend(self.seToPNNs(se))
             else:
-                newList.extend(se)
+                newList.append(se)
```
Comment (ericvaandering) [inline src/python/WMQuality/Emulators/SiteDBClient/SiteDB.py]: Ok, so this was the bug Manoj?

Comment (jha2) [inline src/python/WMQuality/Emulators/SiteDBClient/SiteDB.py]: Yes, this was fixed long time ago.

**Comment by ericvaandering** [comment]
Ok, the last thing to do is to squash these commits into one. 

```
git rebase -i HEAD~9 # last 9 commits
# look up interactive rebasing to understand what you need to do in the editor and how to rewrite a nice commit message
git push -f # once everything looks OK
```

**Comment by jha2** [comment]
Please test it.

**Comment by ericvaandering** [comment]
Merging. Error is expected and just from a failure. Will work on that now.

**Pull request merged**